packages/graphql/tests/schema/rfcs/rfc-003.test.ts

```diff
@@ -0,0 +1,181 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { gql } from "apollo-server";
+import { GraphQLSchema } from "graphql";
+import { Neo4jGraphQL } from "../../../src";
+
+describe("schema/rfs/003", () => {
+    const msg = "Cannot represent null for the relationship Source.targets, try using NonNull operator '!'";
```

Comment:
Could the thrown message possibly be more specific and guiding, for example:

"List type relationship fields must be non-nullable and have non-nullable entries, please change type of `Source.targets` to `[Target!]!`"

We could possibly also add a link to the documentation in this message for background, which leads to me my next question - has this been documented? Perhaps it should be explained in https://neo4j.com/docs/graphql-manual/current/type-definitions/relationships/?

Reply:
Sure!